master/include/logic/checkupManager.hpp

```diff
@@ -32,10 +36,10 @@ class CheckupManager {
     */
     enum class CheckupState {
         WAIT_FOR_ASMS,
-        // START_TOGGLING_WATCHDOG,
```

Comment:
There is not watchdog in 01 still, this is to remain uncommented

Reply:
Yes, but the reasoning behind it was that there is no problem having these states, since the state machine and the state transitions are the same